controllers/campaigns.js

```diff
@@ -0,0 +1,20 @@
+const fs = require("fs")
+const { findAllCampaigns } = require('../models/campaigns');
+const quickStart = require('../services/textToSpeech');
+
+module.exports.getCollection = async (req, res) => {
+  const [data] = await findAllCampaigns();
+  res.json(data);
+};
+
+module.exports.vocalization = async (req, res) => {
+  const fileName = await quickStart(req.body.message);
+res.status(200).send(fileName)
+};
+
+module.exports.playAudio = async (req, res) => {
+const audioFile = `${req.query.audio}.mp3`
+// res.type = path.extname(audio);
+const stream = fs.createReadStream(`/usr/src/app/file-storage/public/${  audioFile}`);
```

Comment:
jamais de chemins absolus, utiliser __dirname

Reply:
et path.join (https://stackoverflow.com/questions/9756567/do-you-need-to-use-path-join-in-node-js)

Reply:
Les chemins ont été modifiés avec path.join et __dirname, c'est mieux ?